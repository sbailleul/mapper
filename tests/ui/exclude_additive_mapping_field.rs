use mapper_impl::Mapper;

#[derive(Mapper)]
pub struct User{
    #[to(Person, exclude, strategy=into)]
    pub name: String
}

pub struct Person{
    pub name: String
}

fn main(){}