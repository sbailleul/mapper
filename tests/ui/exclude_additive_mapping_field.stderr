error: Cannot exclude a field for a destination (Person) not referenced in automatic mapping
 --> tests/ui/exclude_additive_mapping_field.rs:5:18
  |
5 |     #[to(Person, exclude, strategy=into)]
  |                  ^^^^^^^
